### PR TITLE
NP-243 design updates

### PR DESCRIPTION
**Pull request by panosvoudouris**
See ticket for updates included https://citizensadvice.atlassian.net/secure/RapidBoard.jspa?rapidView=159&modal=detail&selectedIssue=NP-243

### CODE REVIEW

File: styleguide/haml_locals.rb [hunk 1]
```diff
@@ -33,10 +33,14 @@
   radio_group_label: 'Select an option',
   radio_group_hint: 'There are two options',
   radio_group_options: ['Option 1', 'Option 2'],
+  radio_has_error: true,
+  radio_error_message: 'Please select a option',
```
Comment (shahzainb) [inline styleguide/haml_locals.rb]: *an option

File: styleguide/haml_locals.rb [hunk 2]
```diff
@@ -33,10 +33,14 @@
...
 
   input_name: 'inputABC',
   input_label: 'This is the label for the input',
   input_hint: 'This is the hint for the input',
+  input_has_error: true,
+  input_error_message: 'This is a error messsage',
```
Comment (shahzainb) [inline styleguide/haml_locals.rb]: *an error?

Comment (panosvoudouris) [reply styleguide/haml_locals.rb]: Yeah yeah yeah

**Comment by soniaturcotte** [comment]
All looks good, but could we get the same bottom margin across the components so they work together on a page? (buttons and forms, I've added to callouts and notices already).

**Comment by panosvoudouris** [comment]
@soniaturcotte done, please check.

**Pull request merged**